tests/models/test_onnx.py

```diff
@@ -24,6 +24,20 @@ def test_model_saves_with_input_sample(tmpdir):
     assert os.path.getsize(file_path) > 3e+06
 
 
+@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
```

Comment:
```suggestion
@pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires GPU machine")
```
you don't need multiple GPUs for this